### ISSUE TITLE
请问现在预训练模型可以用了吗

### ISSUE THREAD

**Issue by luanqi123** [INVOKERer/DeepRFT#40]
如题

**Comment by luanqi123** [comment]
![捕获](https://user-images.githubusercontent.com/76938355/189476203-866fd535-6bdb-4b2b-beba-2cf53d7f28a2.PNG)

**Comment by luanqi123** [comment]
报这个错误

**Comment by INVOKERer** [comment]
你好，模型是可以正常使用的，在这个位置报错应该不是模型的问题（模型应该已经完成了加载），如果你使用的是windows，请尝试将test.py：test_loader  = DataLoader(dataset=test_dataset, batch_size=1, shuffle=False, num_workers=4, drop_last=False, pin_memory=True)中的num_workers设置为0。

**Comment by luanqi123** [comment]
好的，明天我试一下 十分感谢您的解答 祝您生活愉快。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "INVOKERer/DeepRFT"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月10日(星期六) 下午5:01
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [INVOKERer/DeepRFT] 请问现在预训练模型可以用了吗 (Issue #40)





 
你好，模型是可以正常使用的，在这个位置报错应该不是模型的问题（模型应该已经完成了加载），如果你使用的是windows，请尝试将test.py：test_loader  = DataLoader(dataset=test_dataset, batch_size=1, shuffle=False, num_workers=4, drop_last=False, pin_memory=True)中的num_workers设置为0。
